app/models/user.rb

```diff
@@ -1,2 +1,7 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Great job so far but I think it would be great if you add here reference to the class with the foregin key so I recommend to add it here for the three relation since in this three tables there are author_id reference to them 🤩✅ so please add it

###  You can do somthing like this ⬇
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```